BrainPortal/lib/scir.rb

```diff
@@ -74,6 +74,9 @@ def self.cbrain_config
     @config = {
       :extra_qsub_args => rr.cms_extra_qsub_args || "",
       :default_queue   => rr.cms_default_queue   || "",
+      :amazon_ec2_region => rr.amazon_ec2_region || "",
```

Comment:
I don't like that the RemoteResource object now has attributes specifically for amazon cloud stuff... they're supposed to be generic resource objects representing portals and bourreaux...